examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
+    """Plot connectome given a covariance matrix and atlas maps"""
+    imgs = image.iter_img(atlas_maps)
+    regions_coords = np.array([
+        map(np.asscalar, plotting.find_xyz_cut_coords(img)) for img in imgs])
+    np.random.seed(42)
+    node_colors = np.random.rand(len(atlas_maps), 3)
+    plotting.plot_connectome(cov, regions_coords,
+                             nodes_kwargs={'s': 50, 'c': node_colors},
```

Comment:
We don't want to force our users to learn matplotlib for simple things. We need to start from the idea that our base users do not know anything about matplotlib.

Sent from my phone. Please forgive brevity and mis spelling



On Feb 20, 2015, 09:56, at 09:56, Alexandre Abraham <notifications@github.com> wrote:
>>  from nilearn.plotting import cm
>>  
>>  
>> +def plot_connectome(cov, atlas_maps, **kwargs):
>> +    """Plot connectome given a covariance matrix and atlas maps"""
>> +    imgs = image.iter_img(atlas_maps)
>> +    regions_coords = np.array([
>> +        map(np.asscalar, plotting.find_xyz_cut_coords(img)) for img
>in imgs])
>> +    np.random.seed(42)
>> +    node_colors = np.random.rand(len(atlas_maps), 3)
>> +    plotting.plot_connectome(cov, regions_coords,
>> +                             nodes_kwargs={'s': 50, 'c':
>node_colors},
>
>`s` and `c` are matplotlib standards. This is the first thing I would
>try.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/nilearn/nilearn/pull/441/files#r25056268